### PR TITLE
Add Sentry calls  to monitor for token issues

### PR DESCRIPTION
**Pull request by ghost**
## Description
Adds sentry logging per: department-of-veterans-affairs/covid19-chatbot#186

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
N/A

### CODE REVIEW

**Comment by karlbrown-va** [comment]
shall we log to the console  at the same time? to make it easier for devs experiencing it to see the issue right away?

**Comment by shaunhubbard-va** [comment]
usually `console` is a violation, if vfs ok with it I would say yes.

File: src/applications/coronavirus-chatbot/createCoronavirusChatbot.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ export default (_store, widgetType) => {
       });
       window.WebChat.renderWebChat(webchatOptions, root);
     } catch (err) {
+      Sentry.captureException(err);
```
Comment (ncksllvn) [inline src/applications/coronavirus-chatbot/createCoronavirusChatbot.js:28]: It's weird to me that below this there are two different error events logged to GA...should one of them be in a different place in the code?

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: src/applications/coronavirus-chatbot/createCoronavirusChatbot.js (hunk 1 above)
Comment (shaunhubbard-va) [reply src/applications/coronavirus-chatbot/createCoronavirusChatbot.js:28]: I have the same thoughts, but this was lower on the priority bucket for us. I would think we need to understand why we put two in here before changing them.

**Review by mchelen-gov — COMMENTED** [review]
would it help to record any message or scope such as:
- https://github.com/department-of-veterans-affairs/vets-website/blob/f6540d714ee8654258ecf99d0fe174c95f1b93ba/src/platform/forms/save-in-progress/api.js#L30
- https://github.com/department-of-veterans-affairs/vets-website/blob/ddabad2c4a605f17a4b29c14e1384a30225e06b4/src/platform/forms/save-in-progress/actions.js#L303

File: src/applications/coronavirus-chatbot/createCoronavirusChatbot.js (hunk 1 above)
Comment (mchelen-gov) [reply src/applications/coronavirus-chatbot/createCoronavirusChatbot.js:28]: @shaunhubbard-va is there an ticket created to address that?

**Comment by shaunhubbard-va** [comment]
@mchelen-gov I dont think it would hurt, I have not done sentry in quite some time so I am not up to speed on best practices

**Review by mchelen-gov — APPROVED** [review]

File: src/applications/coronavirus-chatbot/createCoronavirusChatbot.js (hunk 1 above)
Comment (karlbrown-va) [reply src/applications/coronavirus-chatbot/createCoronavirusChatbot.js:28]: this was the relevant thread:
https://dsva.slack.com/archives/C011BG82ZL6/p1587498673442600

But I'm not sure why two events are triggered nonetheless.

Comment (shaunhubbard-va) [reply src/applications/coronavirus-chatbot/createCoronavirusChatbot.js:28]: I would like to know how events are created systematically if possible.

**Comment by mchelen-gov** [comment]
@shaunhubbard-va yeah same, i was just poking around the codebase to see what is used in other places. maybe we can see how this comes through in sentry and decide whether to add more info later

**Comment by shaunhubbard-va** [comment]
I would think thats best in the Interim I will read docs on Sentry/JS for later.

**Review by rianfowler — APPROVED** [review]

**Pull request merged**